Wikipedia/UI-V5/WMFSearchViewController.m

```diff
@@ -410,6 +499,23 @@ - (IBAction)searchForSuggestion:(id)sender {
     [self searchForSearchTerm:self.searchField.text];
 }
 
+- (IBAction)setLanguageOne:(id)sender {
+    [self selectLanguageForButton:sender];
+}
+
+- (IBAction)setLanguageTwo:(id)sender {
+    [self selectLanguageForButton:sender];
+}
+
+- (IBAction)setLanguageThree:(id)sender {
```

Comment:
IIUC each method is doing the same thing: call `selectLanguageForButton:` with the `sender`, which seems to me like you could just use `selectLanguageForButton:` as the IBAction